content/Software Support/Upload/If-there-are-Unknown-devices-in-the-board-selector-and-ports-without-a-board-name-in-the-Tools-Port-menu.md

```diff
@@ -0,0 +1,100 @@
+---
+title: "If there are \"Unknown\" devices in the board selector and ports without a board name in the Tools > Port menu"
```

Comment:
```suggestion
title: "If there are \"Unknown\" devices in the board selector or ports without a board name in the Tools > Port menu"
```
I think "or" fits here. As long as one condition is true, the article applies. This is also consistent with the language in the first paragraph:

> Learn how to troubleshoot and resolve cases where the Arduino IDE shows "Unknown" devices **or** fails to assign a board name in the board selector and _Tools > Port_ menu.